################
# continuous recording
continuous_topics: [/tf, /joint_states]

################
# triggered recording
trigger_topics: [/stereo/left/image_color, /stereo/right/image_color, /stereo/left/camera_info, /stereo/right/camera_info, /cam3d/rgb/points, /map, /scan_front, /scan_rear, /scan_top]

# setup time trigger
trigger_timestep: 10

# setup tf triggers
trigger_record_translation: 0.1
trigger_record_rotation: 0.1
wanted_tfs:
 - reference_frame: /map
   target_frame: /base_link
   trigger_record_translation: 0.1
   trigger_record_rotation: 0.2
